src/catalog/src/durable/initialize.rs

```diff
@@ -68,6 +68,13 @@ pub(crate) const ENABLE_0DT_DEPLOYMENT: &str = "enable_0dt_deployment";
 /// NOTE: Weird prefix because we can't start with a `0`.
 pub(crate) const WITH_0DT_DEPLOYMENT_MAX_WAIT: &str = "with_0dt_deployment_max_wait";
 
+/// The key used within the "config" collection where we store a mirror of the
+/// `0dt_deployment_panic_after_timeout` "system var" value. This is mirrored so
+/// that we can toggle the flag with LaunchDarkly, but use it in boot before
+/// LaunchDarkly is available.
+pub(crate) const WITH_0DT_DEPLOYMENT_PANIC_AFTER_TIMEOUT: &str =
+    "0dt_deployment_panic_after_timeout";
```

Comment:
The name is also different here from `0dt_panic_after_timeout`. Maybe this is the reason that it's not working locally for me. I'll try to push a commit to unify the names.

Reply:
Actually based on the comment maybe this is intentional.